bin/board.rb

```diff
@@ -0,0 +1,12 @@
+class Board
+  attr_accessor :grid
```

Comment:
Just make this `attr_reader`. Here are some pointers to determine whether we should use attr_accessor vs attr_reader (or we even need to put attributes in this line in the first place):

1. Determine whether you need an attribute to be exposed to other objects (in this case do you need @grid to be used by other objects (i.e will the game or player object need this attribute from board? if so then yeah you need to place it in this line)
2. If this attribute needs to be exposed to another object as a public interface then next question is, do those other objects need to just read off from this attribute (in that case `attr_reader` is enough) or do those other object need to modify this attribute (in this case you need `attr_accessor`)? As much as possible you don't want to expose your object's attributes to other objects if not necessary. Also by default use `attr_reader` and find ways for other objects to not have to modify this object's attributes directly. 
3. One clarification in the case of array attributes (such as this grid). If you need another object to push to or pop from this array attribute then you can still just put it in `attr_reader`. Pushing or popping into an array doesn't constitute 'modifying' it in this context. Modifying an array means reassigning its value (ex: you want to reassign grid into another value like arr = "I am a new value here")